backend/scripts/run_insert_photos.sh

```
@@ -0,0 +1,31 @@
+if [ -z "$1" ]; then
+    echo "Error: No user_id provided."
+    echo "Usage: ./run_insert_photos.sh <user_id>"
+    exit 1
+fi
+
+USER_ID=$1
+
+echo "Starting photo insertion for user: $USER_ID"
+
+# Change to backend directory to ensure environment variables and imports work
+cd "$BACKEND_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)